_data/news.yml

```diff
@@ -194,3 +194,7 @@
   date: 2024-10-24
   linked_pr: 1526
   description: The page is enriched with additional links and a bibliography. [Discover the page here](human_pathogen_genomics).
+- name: "New page: Data workflow diagram"
```

Comment:
@bedroesb can you kindly update page name here after "News page"? The new name is "Creating Data Workflow"